standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -421,14 +425,14 @@ public String getZkUrl() {
     @Synchronized
     public void close() throws Exception {
         if (isInProcSegmentStore) {
-            for ( ServiceStarter starter : this.nodeServiceStarter ) {
+            for (ServiceStarter starter : this.nodeServiceStarter) {
                 starter.shutdown();
             }
         }
         if (isInProcController) {
-            for ( ControllerServiceMain controller : this.controllerServers ) {
-                    controller.stopAsync();
-                }
+            for (ControllerServiceMain controller : this.controllerServers) {
```

Comment:
most of the changes in this file look unrelated. Any chance you could revert them?

Reply:
my bad. 
done